src/LabelNumeral.jl

```diff
@@ -55,6 +55,6 @@ isless(n1::LabelNumeral, n2::LabelNumeral) = getval(n1) < getval(n2)
 ## Arithmetic
 # Multiple argument operators
 for op in [:+, :-,:max, :min]
-    @eval ($op){T <: Integer}(ns::LabelNumeral{T}...) =
-        $(op)(map(n -> getval(n), ns)...) |> LabelNumeral{T}
+    @eval ($op){T <: Integer}(n1::LabelNumeral{T}, ns::LabelNumeral{T}...) =
+        $(op)(map(getval, n1, ns)...) |> LabelNumeral{T}
```

Comment:
I may like to suggest all these operators as binary or higher order operators. So may be 2 params should be made mandatory. Unlike, normal numbers these are mostly labels. So unary operators as +, - do not have any practical sense. Also, max and min for one number though is correct in a set theoretic sense but do not have much of significance.